packages/frontend-api/src/Model/Store/OpeningHours/StoreOpeningHoursApiProvider.php

```diff
@@ -12,6 +12,11 @@
 
 class StoreOpeningHoursApiProvider
 {
+    public const STATUS_OPEN = 'OPEN';
+    public const STATUS_CLOSED = 'CLOSED';
+    public const STATUS_OPEN_SOON = 'OPEN_SOON';
+    public const STATUS_CLOSED_SOON = 'CLOSED_SOON';
```

Comment:
missing strict type hint. Also, please consider moving the constants into the pseudo-enum class (see eg. https://github.com/shopsys/shopsys/blob/16.0/packages/framework/src/Model/Order/Item/OrderItemTypeEnum.php)

Reply:
fixed